src/api/Main/PresentationLayer/Shrooms.API/Controllers/Kudos/KudosController.cs

```diff
@@ -150,6 +151,14 @@ public IEnumerable<KudosTypeViewModel> GetKudosTypes()
             return kudosTypeViewModel;
         }
 
+        [HttpGet]
+        [PermissionAuthorize(Permission = BasicPermissions.Kudos)]
+        public KudosTypeViewModel GetSendType()
```

Comment:
Please keep consistent naming. If it's `_kudosService.GetKudosTypeSend`, then why controller endpoint named in different pattern? Also if it's more natural to say `GetSendType`, then perhaps in service it also could be named - `GetSendKudosType`, and then controller endpoint accordingly.